Tests/Request/ParamFetcherTest.php

```diff
@@ -83,6 +83,21 @@ public function setup()
         $this->container = $this->getMockBuilder('Symfony\Component\DependencyInjection\ContainerInterface')->getMock();
     }
 
+    public function testConstructorCallWithNullValidatorShouldTriggerDeprecation()
```

Comment:
We can make use of the `@expectedDeprecation` feature of the PHPUnit bridge:

```php
/**
 * @expectedDeprecation put the deprecation message here
 * @group legacy
 */
public function testConstructorCallWithNullValidatorShouldTriggerDeprecation()
{
    new ParamFetcher($this->container, $this->paramReader, $this->requestStack, null);
}
```

Reply:
Great ! I didn't know `expectedDeprecation` annotation, thank you :))